upload_run.py

```diff
@@ -78,6 +78,15 @@
                              help='Override "parser" in config file. '
                                   'Choose the type of sequencer data that is being uploaded. '
                                   'Supported parsers: ' + str(supported_parsers))
+argument_parser.add_argument('-m', '--multithreading',
+                             action='store_true',  # This line makes it not parse a variable
+                             help='!!EXPERIMENTAL FEATURE!! '
+                                  'Advanced User Feature. Upload will be spread across multiple threads. '
+                                  'If the config file is set to False, this will override it to True')
+argument_parser.add_argument('-t', '--threads', action='store',
+                             help='!!EXPERIMENTAL FEATURE!! To be used with the -m argument. '
+                                  'The number of threads to use when multithreading is enabled. '
+                                  'Minimum threads = 2, Recommended maximum threads = 8')
```

Comment:
I believe our discussion was about how to handle it from the config file side, as it will mostly be an admin feature.

I could change command line such that we only have the `--threads` option, and inputting anything enables multithreading (only allowing values >=2) which would probably be a cleaner way of handling it on the user side.

Alternatively we could drop support for multithreading from command line args altogether and have it only accessible via the config file.

Reply:
I like having the command line option so I'd say keep that.  But having the 2 options required for the same feature is what's throwing me off.  

I see below where you're checking 
```python
if args.multithreading is True
``` 
I'd suggest changing it to 
```python
if args.threads > 1
```
That way you can just remove the `-m` option.  I could reasonably see someone setting `--threads 1` if they'd been testing with more, and getting annoyed that they have to remove the arg.